config/samples/extra-roles.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: external-dns-operator
+  namespace: external-dns
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+      - secrets
+      - namespaces
```

Comment:
Namespaces too? In the local role?

Reply:
Yes! otherwise the operator logs show errors on get namespace external-dns, so I had to keep it

Reply:
I think this needs to re-open.  Seems fair to "get", "watch" on the namespace to squelch the error message, but you do not want to grant permissions to update it.